build_projects/dotnet-host-build/build.sh

```diff
@@ -93,8 +93,8 @@ done < "$REPOROOT/branchinfo.txt"
 [ -z "$DOTNET_INSTALL_DIR" ] && export DOTNET_INSTALL_DIR=$REPOROOT/.dotnet_stage0/$(uname)
 [ -d "$DOTNET_INSTALL_DIR" ] || mkdir -p $DOTNET_INSTALL_DIR
 
-
-$REPOROOT/scripts/obtain/dotnet-install.sh --channel preview
+DOTNET_INSTALL_SCRIPT_URL="https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.sh"
+curl -sSL "$DOTNET_INSTALL_SCRIPT_URL" | bash /dev/stdin --channel rel-1.0.0 --verbose
```

Comment:
We should be keeping the non-Windows and the Windows scripts in line as much as possible.  We shouldn't get out of sync between what windows does and what non-Windows does.

https://github.com/dotnet/core-setup/blob/release/1.1.0/build_projects/dotnet-host-build/build.ps1#L104